syntax = "proto3";

package sekai;

message EventId {
  optional int32 event_id   = 1;
  optional int32 chapter_id = 2;
}
